torch/_dynamo/symbolic_convert.py

```diff
@@ -672,6 +672,8 @@ def skip(v: VariableTracker):
     def replace_all(self, oldvar: VariableTracker, newvar: VariableTracker):
         if isinstance(oldvar.mutable_local, side_effects.MutableSideEffects):
             newvar = self.output.side_effects.mutation(oldvar, newvar)
+        elif isinstance(oldvar.mutable_local, side_effects.AttributeMutationExisting):
+            newvar = oldvar
```

Comment:
What's going on here?

Reply:
You still need to explain what's going on here

Reply:
We replace_all with a tensor that is stored as an attribute mutation - we don't need a clone, we just need to new to old.

Reply:
I'm also not sure what this is doing

Reply:
sec